src/index.js

```diff
@@ -1,59 +1,33 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import Payment from 'payment';
+import creditCardType from 'credit-card-type';
+import luhn from 'luhn';
+
+import { configure } from './config';
+import { sanitizeNumber } from './utils';
 
 class ReactCreditCards extends React.Component {
   constructor(props) {
     super(props);
 
-    this.setCards();
+    const validCardTypes = configure();
+
+    this.state = {
+      validCardTypes,
+    };
   }
 
-  static propTypes = {
```

Comment:
Why did you move the propTypes/defaultProps from the class declaration? ESLint?

Reply:
Yeah, probably had some global config that was warning so I moved them to get rid of it while working and forgot to move it back after finishing, will keep this comment to remember to do that before merging this PR.

Reply:
Nah, that's alright